djangocms_admin_style/sass/components/_forms.scss

```diff
@@ -737,6 +737,7 @@ form select[multiple] {
 
 //related-widget
 .related-widget-wrapper {
+    overflow: visible;
```

Comment:
@lory87 I would recommend for the future to always add comments like:

``/* required in /cms/pages... to ensure outline is not cut off */``